tests/run-test-environment

```diff
@@ -56,13 +56,14 @@ shift
 
 
 [ "$TEST_TYPE" == "acceptance" ] && {
-    COMPOSE_CMD="docker-compose -p acceptance-tests -f $INTEGRATION_PATH/docker-compose.yml"
+    COMPOSE_CMD="docker-compose -p acceptance-tests -f $INTEGRATION_PATH/docker-compose.yml -f $INTEGRATION_PATH/docker-compose.client.yml -f $INTEGRATION_PATH/docker-compose.demo.yml"
```

Comment:
are you sure we need to start client in acceptance tests? IIRC the test emulate a client so we don't need to whole image & vm here